### ISSUE TITLE
UI - Design/Input - Where do new Professional Skills, Software Skills, and Coding Languages get Added?

### ISSUE THREAD

**Issue by sethborne** [RecursiveThinking/recursive_thinking_website#35]
So in the Edit profile, there are categories for Professional Skills, Software Skills and Coding Languages.

Where do these get added?  The input prompt is always Search - so should the User be able to Add New Items to any of these inputs if they Search and (still) haven't found what they are looking for?

**Comment by MynockSpit** [comment]
I was imagining it'd be a dropdown list and if you clicked on one it'd add it to your profile. I think LinkedIn does something similar?

**Comment by hdskelcher** [comment]
@sethborne @MynockSpit Yes, I was thinking exactly like LinkedIn: the user would start typing - the options would pop down, allowing them to select and if what they are looking for is not on the list they could type the whole thing and add a new item
